### PR TITLE
CAIP-10 Proposed Changes

### PR DESCRIPTION
**Pull request by pedrouid**
As discussed on CASA Meeting #1, it reached consensus that CAIP-10 should change endianness as proposed on Issue #51

### CODE REVIEW

File: CAIPs/caip-10.md [hunk 1]
```diff
@@ -24,29 +24,29 @@ The motivation for proposal stem from designing a chain-agnostic protocol for co
 
 ## Specification
 
-The account id specification will be prefixed with the CAIP-2 blockchain ID and delimited with an at sign (`@`) commonly used to define addresses.
+The account id specification will be prefixed with the CAIP-2 blockchain ID and delimited with a comma sign (`:`)
```
Comment (ligi) [inline CAIPs/caip-10.md:27]: I think you mean colon here

**Review by ligi — CHANGES_REQUESTED** [review]

**Review by oed — CHANGES_REQUESTED** [review]
Please include information about the legacy format!

File: CAIPs/caip-10.md [hunk 2]
```diff
@@ -24,50 +24,54 @@ The motivation for proposal stem from designing a chain-agnostic protocol for co
...
 The goals of the general account ID format is:
 
 - Uniqueness between chains regardless if they are mainnet or testnet
-- Readibility using `@` to easily identify the account address
+- Readibility using the prefix of a chainId to quickly identify before parsing the address
 - Restricted to constrained set of characters and length for parsing
 
+## Backwards Compatibility
+
+Previously CAIP-10 schema was defined by appending as suffix the CAIP-2 chainId delimited by the at sign (`@`)
```
Comment (pedrouid) [inline CAIPs/caip-10.md:54]: I've added this section to comment on the previous schema. cc @oed

Comment (oed) [reply CAIPs/caip-10.md:54]: I think an example would be good here to remove any ambiguity.

Comment (pedrouid) [reply CAIPs/caip-10.md]: I've moved the "backwards compatibility" section below the test cases and added a legacy example

**Review by oed — APPROVED** [review]
lgtm

**Review by ligi — APPROVED** [review]
LGTM

**Pull request merged**